README.md

```diff
@@ -27,7 +27,18 @@ This plugin uses familiar Kubernetes and container tools to bring you a rich Kub
 
 ## Installing the plugin into your IDE
 
-(coming soon)
+If you'd like to try out the plugin pre-release, you can build it from source and install it into your IDE:
+
+1) Clone this repository to your machine:<br />
+   `git clone git@github.com:GoogleContainerTools/google-container-tools-intellij.git`
+2) Build the plugin from the root of the repository:<br />
+   `./gradlew buildPlugin`
+3) Find the `.zip` binary in the root of the repository under: <br />
+   `build/distributions/google-container-tools-intellij.zip`
+4) Install the plugin into your JetBrains IDE:
+   1) Go to `Preferences > Plugins`
```

Comment:
Mention that this menu is named `Settings` on non-Mac OS, like Linux?

Reply:
I'm not on my linux machine now: is it called Settings instead of Preferences in that case?

Reply:
It is, yes.